session/txn.go

```diff
@@ -488,6 +488,7 @@ type txnFuture struct {
 
 func (tf *txnFuture) wait() (kv.Transaction, error) {
 	startTS, err := tf.future.Wait()
+	failpoint.Inject("txnFutureWait", func() {})
```

Comment:
Unused code?

Reply:
It's prepared to be used in external integration tests